setup/updating.md

```diff
@@ -65,8 +65,54 @@ To update Coder, follow these steps:
    ```console
    helm upgrade --namespace coder --install --atomic --wait \
      --version 1.16.1 coder coder/coder --values current-values.yaml
+
+   ```
+
+1. If you're upgrading from v1.20 to v1.21,
+   [update your Helm chart](../guides/admin/helm-charts.md) via the steps below.
+   Refer to the `1.21`
+   [helm chart](https://github.com/cdr/enterprise-helm/blob/main/values.yaml)
+   when making such changes:
+
+   Rename all instances of `cemanager` to `coderd`.
+
+   Redirect all traffic from your custom ingress to `coderd`.
+
+   `coderd` now serves TLS certificates; to upgrade the NGINX ingress
+   controller, set `coderd.serviceNext` to `true`. (See our
```

Comment:
I am pretty sure this helm value actually removes the ingress controller and serves Coder directly through a LoadBalancer. cc @kylecarbs to confirm/deny.

Reply:
If that's the case, we need to update the messaging to people when they upgrade: https://github.com/cdr/enterprise-helm/blob/main/templates/NOTES.txt#L51